packages/common/src/GRPCTypeTranslation.ts

```diff
@@ -1364,6 +1364,9 @@ export function pendingUpdate(
             };
         case undefined:
             throw Error('Unexpected missing pending update');
+        default:
+            // TODO support new updates
+            throw Error('Unsupported update: ' + effect.oneofKind);
```

Comment:
Was there an issue where this came up?

Reply:
Without this, I now get type errors, because I don't handle all possible cases, after the new consensus update types have been added to the gRPC API.

Reply:
thanks.